### PR TITLE
Dockerization of comclassroomweb

### PR DESCRIPTION
**Pull request by divyanshu-parihar**
added dockerfiles with main service named comclassroomweb and supportive service mongo

### CODE REVIEW

File: config/test.yml [hunk 1]
```diff
@@ -3,4 +3,4 @@ app:
 
 
 database:
-  string: 'mongodb://127.0.0.1:27017/community-classrooms'
+  string: "mongodb://127.0.0.1:27017/community-classrooms"
```
Comment (YashKumarVerma) [inline config/test.yml:6]: please don't change the config files, single quotes return the data "as it is". 

https://stackoverflow.com/questions/19109912/yaml-do-i-need-quotes-for-strings-in-yaml

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+version: "2"
+services:
+  app:
+    container_name: comclassroomweb
```
Comment (YashKumarVerma) [inline docker-compose.yml:4]: please mention backend here, might be confusing for some devs

File: src/services/database/index.js [hunk 3]
```diff
@@ -11,6 +11,8 @@ const databaseConfigurations = {
  * Initializes MongoDB connection using configs and logs status
  */
 const initializeMongoDB = () => {
+  console.log(config.has('database.string'));
+  // console.log(config.get('database.string'))
```
Comment (YashKumarVerma) [inline src/services/database/index.js:15]: why are we doing this?

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]
also, we are using npm, not yarn. please do not commit the yarn.lock file

**Comment by divyanshu-parihar** [comment]
made some changes

**Review by YashKumarVerma — APPROVED** [review]

**Comment by YashKumarVerma** [comment]
thanks !

**Pull request merged**